python/dask_cudf/dask_cudf/core.py

```diff
@@ -280,6 +281,7 @@ def var(
         split_every=False,
         dtype=None,
         out=None,
+        naive=False,
```

Comment:
We can remove the `naive` option once we can confirm that the new approach is more stable **and** avoids any performance  degradation (so far, it seem performance is better anyway).